sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1285,24 +1282,23 @@ private synchronized void updateReleaseDetails(ReleaseDetails releaseDetails) {
     /**
      * Get reporting parameters for updated release.
      *
-     * @param isPublic            are the parameters for public group or not.
-     *                            For public group we report install_id and release_id.
-     *                            For private group we report distribution_group_id and release_id.
-     * @param distributionGroupId distribution group id.
+     * @param isPublic are the parameters for public group or not.
+     *                 For public group we report install_id, distribution_group_id and release_id.
+     *                 For private group we report distribution_group_id and release_id.
      */
     @NonNull
-    private String getReportingParametersForUpdatedRelease(boolean isPublic, String distributionGroupId) {
+    private String getReportingParametersForUpdatedRelease(boolean isPublic) {
         String reportingParameters = "";
         AppCenterLog.debug(LOG_TAG, "Check if we need to report release installation..");
         String lastDownloadedReleaseHash = SharedPreferencesManager.getString(PREFERENCE_KEY_DOWNLOADED_RELEASE_HASH);
+        String distributionGroupId = SharedPreferencesManager.getString(PREFERENCE_KEY_DISTRIBUTION_GROUP_ID);
```

Comment:
read it from the parameter like before, that will avoid an additional read from storage